### PR TITLE
Update hooks - reset progress when playback stops

### PR DESCRIPTION
**Pull request by alangumer**

### CODE REVIEW

**Comment by alangumer** [comment]
@dcvz can you please include this in the latest tag 2.0.0-rc13?

File: src/hooks.ts [hunk 1]
```diff
@@ -83,6 +83,10 @@ export function useProgress(updateInterval?: number) {
   }
 
   useEffect(() => {
+    if (playerState === State.Stopped) {
+      setState({ position: 0, duration: 0, buffered: 0 });
+      return;
+    }
```
Comment (Guichaguri) [inline src/hooks.ts:89]: It's not the desirable solution, but it does fixes the issue pretty well

**Comment by curiousdustin** [comment]
Was there an issue ticket associated with this? I’m not seeing it. Should a similar fix be applied to dev?

**Comment by Guichaguri** [comment]
This is probably seen as a breaking change, but I think it will be superseded by #935

**Pull request merged**